Core.ApplicationServices/Core.ApplicationServices.csproj

```diff
@@ -103,6 +103,7 @@
     <Compile Include="Interface\ExhibitUsage\InterfaceExhibitUsageService.cs" />
     <Compile Include="Interface\Usage\IInterfaceUsageService.cs" />
     <Compile Include="Interface\Usage\InterfaceUsageService.cs" />
+    <Compile Include="Model\System\DeleteResult.cs" />
```

Comment:
SystemDeleteResult var OK, for når man ser koden andre steder, ser du ikke namespace context.